plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -58,6 +58,7 @@
     ]]>
     </change-notes>
 
+    <depends optional="true" config-file="go.xml">org.jetbrains.plugins.go</depends>
```

Comment:
Let's check that everything works in GoLand

Reply:
Everything works